scripts/docker/image-runner.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2024 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+docker run \
+    --privileged \
+    -d \
+    --net host \
```

Comment:
It would be able to use default network with port forwarding, instead of using same network as its host.

Reply:
Resolving this with using `-p 8080:8080` instead. Let's see it works over LAB network.